### ISSUE TITLE
FolioGraphqlClient OKComputer check is lying to us

### ISSUE THREAD

**Issue by corylown** [sul-dlss/mylibrary#985]
```
3.2.0 :012 > FolioGraphqlClient.new.ping
 => #<HTTP::Response/1.1 503 Service Unavailable {"Date"=>"Tue, 17 Oct 2023 14:18:04 GMT", "Server"=>"Apache/2.4.41 (Ubuntu)", "Strict-Transport-Security"=>"max-age=31536000; includeSubDomains", "Content-Length"=>"299", "Connection"=>"close", "Content-Type"=>"text/html; charset=iso-8859-1"}> 
3.2.0 :013 > GraphqlCheck.new.check
 => "Connected to Folio GraphQL" 
```

**Comment by hudajkhan** [comment]
We should figure out if this should get fixed.

**Comment by corylown** [comment]
This might have been fixed by: https://github.com/sul-dlss/mylibrary/pull/987

**Comment by taylor-steve** [comment]
Confirmed this appears to be fixed by #987.

Without #987:
```
irb(main):001> FolioGraphqlClient.new.ping
  [HTTP] > GET https://httpstat.us/503/graphql?query=%7B__typename%7D
  [HTTP] < 503 Service Unavailable application/json (290.012ms)
=> #<HTTP::Response/1.1 503 Service Unavailable {"Content-Length"=>"48", "Connection"=>"close", "Content-Type"=>"application/json", "Date"=>"Thu, 15 Feb 2024 17:43:15 GMT", "Server"=>"Kestrel", "Set-Cookie"=>["ARRAffinity=fab4dbbced8c587be327dc4652709304d9a096a7c035b45ace6409c8bb4aca8e;Path=/;HttpOnly;Secure;Domain=httpstat.us", "ARRAffinitySameSite=fab4dbbced8c587be327dc4652709304d9a096a7c035b45ace6409c8bb4aca8e;Path=/;HttpOnly;SameSite=None;Secure;Domain=httpstat.us"], "Strict-Transport-Security"=>"max-age=2592000", "Request-Context"=>"appId=cid-v1:3548b0f5-7f75-492f-82bb-b6eb0e864e53"}>
irb(main):002> GraphqlCheck.new.check
  [HTTP] > GET https://httpstat.us/503/graphql?query=%7B__typename%7D
  [HTTP] < 503 Service Unavailable application/json (292.982ms)
=> "Connected to Folio GraphQL"
```

With #987:
```
irb(main):001> FolioGraphqlClient.new.ping
  [HTTP] > GET https://httpstat.us/503/graphql?query=%7B__typename%7D
  [HTTP] < 503 Service Unavailable application/json (298.149ms)
=> false
irb(main):002> GraphqlCheck.new.check
  [HTTP] > GET https://httpstat.us/503/graphql?query=%7B__typename%7D
  [HTTP] < 503 Service Unavailable application/json (339.429ms)
=> "Unable to connect to Folio GraphQL"
```
